Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -761,8 +761,11 @@ public void HandleReply(CallbackQuery query)
 
                 if (player.CurrentQuestion.QType == QuestionType.Lynch)
                 {
-                    var msg = GetLocaleString("PlayerVotedLynch", player.GetName(), target.GetName());
-                    SendWithQueue(msg);
+                    if (DbGroup.EnableSecretLynch == false)
+                    {
+                        var msg = GetLocaleString("PlayerVotedLynch", player.GetName(), target.GetName());
```

Comment:
Also, this logic is backwards.  If secret lynched is enabled ("Allow"), then this value will be true, not false.